apps/blues-wireless/src/note_c_hooks.c

```diff
@@ -0,0 +1,154 @@
+/**
+ #
+ # Licensed to the Apache Software Foundation (ASF) under one
+ # or more contributor license agreements.  See the NOTICE file
+ # distributed with this work for additional information
+ # regarding copyright ownership.  The ASF licenses this file
+ # to you under the Apache License, Version 2.0 (the
+ # "License"); you may not use this file except in compliance
+ # with the License.  You may obtain a copy of the License at
+ #
+ #  http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing,
+ # software distributed under the License is distributed on an
+ # "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ # KIND, either express or implied.  See the License for the
+ # specific language governing permissions and limitations
+ # under the License.
+ #
+ */
+
+
+#include "note_c_hooks.h"
+#include "bsp/bsp.h"
+
+#include "hal/hal_i2c.h"
+
+#include "os/os_time.h"
+
+#include <console/console.h>
+
+static const size_t REQUEST_HEADER_SIZE = 2;
+
+const uint8_t i2c_num = 0;
+bool i2c_initialized = false;
+
+
+uint32_t
+platform_millis(void)
+{
+    return (uint32_t)(os_get_uptime_usec()/1000);
+}
+
+void
+platform_delay(uint32_t ms)
+{
+    os_time_delay((OS_TICKS_PER_SEC/1000) * ms);
```

Comment:
If OS_TICKS_PER_SEC is less then 1000 this will always call os_time_delay() with 0.
OS_TICK_PER_SEC is often set to 128 for bluetooth devices

Reply:
maybe `os_time_ms_to_ticks32()` could be used for ticks calculation

Reply:
Ok. I will change this.

Reply:
This will work...
```
void
platform_delay(uint32_t ms)
{
    os_time_t d_time;
    d_time = os_time_ms_to_ticks32(ms);
    os_time_delay((d_time);
}
```